dotnet/src/SemanticKernel.Abstractions/AI/ChatCompletion/ChatHistory.cs

```diff
@@ -60,6 +60,16 @@ public ChatHistory(IEnumerable<ChatMessageContent> messages)
     public void AddMessage(AuthorRole authorRole, string content, Encoding? encoding = null, IReadOnlyDictionary<string, object?>? metadata = null) =>
         this.Add(new ChatMessageContent(authorRole, content, null, null, encoding, metadata));
 
+    /// <summary>
+    /// <param name="name">Name of the author of the message</param>
+    /// <param name="authorRole">Role of the message author</param>
+    /// <param name="content">Message content</param>
+    /// <param name="encoding">Encoding of the message content</param>
+    /// <param name="metadata">Dictionary for any additional metadata</param>
+    /// </summary>
+    public void AddMessage(string? name, AuthorRole authorRole, string content, Encoding? encoding = null, IReadOnlyDictionary<string, object?>? metadata = null) =>
```

Comment:
`ChatHistory` is out generic abstraction that is AI model agnostic.

Is the `Name` property support widely among chat completion models?

If not, we should consider adding this to the metadata as an optional property. Developers can create their own extension methods to make this more convenient in the future.